### PR TITLE
coloured heading

### LINKED ISSUE #4: Create a stylesheet and link it in
Notes:
- create `styles` folder
- name the stylesheet `styles.css` and place in the `styles folder`
- add a background
- add a header with some intro text

### PR DESCRIPTION
**Pull request by nettle790**
added a colour to the heading.

### CODE REVIEW

**Review by codeTrackLift — CHANGES_REQUESTED** [review]
Please follow notes as outlined in Issue #4 

create styles folder
name the stylesheet styles.css and place in the styles folder
add a background
add a header with some intro text

**Comment by nettle790** [comment]
issue #4 

please review the changes

styles folder created within which styles.css file has been created which is linked to index.html.

applied a background colour and an  image.

provided a header

File: index.html [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-    <body>
-        <h1>Hello World!</h1>
-    </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="styles/styles.css" type="text/css">
```
Comment (codeTrackLift) [inline index.html]: suggest to prefix url w `./`

is the `type="text/css"` necessary?

File: index.html [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+</head>
+
+<body>
+  <h1 style="color:red;">Hello World!</h1>
+  <h2>ATHENA the patron goddess of Greece<h2><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br></br>
```
Comment (codeTrackLift) [inline index.html]: Any reason for the `<br>`s?

If extra space is needed, suggest to apply styling via added stylesheet.

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: pink;
+  text-align: center;
+  background-image: url("athens0.jpeg");
```
Comment (codeTrackLift) [inline styles/styles.css]: suggest to create an `images` folder in project `root` then move image there and re-route.

also suggest to prefix url with `./` similar to href

**Comment by codeTrackLift** [comment]
Nice job w the changes, please see suggestions/comments.  Let me know w any questions, thanks!

**Comment by codeTrackLift** [comment]
@nettle790 also friendly reminder about project tracking and keeping the project board updated.

Please review the [Project Board](https://github.com/orgs/codepharm/projects/6/views/1), you'll see that I've updated the related issue status by moving it to the In Review swim lane, and linked this as the related PR.

I've also assigned this to you as the PR `assignee`, and added this to the project board (see right sidebar of this PR).  These practices are how we stay synchronized in an asynchronous work environment. 

Hope that makes sense, let me know with any questions.  Thanks!

**Comment by nettle790** [comment]
Thanks.

File: index.html (hunk 1 above)
Comment (codeTrackLift) [reply index.html]: @nettle790

File: index.html (hunk 2 above)
Comment (codeTrackLift) [reply index.html]: @nettle790

File: styles/styles.css (hunk 3 above)
Comment (codeTrackLift) [reply styles/styles.css]: @nettle790

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: pink;
+  text-align: center;
+  background-image: url("./images/athens0.jpeg");
```
Comment (nettle790) [inline styles/styles.css]: url file path changed to ./

File: index.html (hunk 1 above)
Comment (codeTrackLift) [reply index.html]: Please prefix href with `./` in line 9

**Review by codeTrackLift — COMMENTED** [review]
Please move `images` folder out of `styles` folder, such that it is a sibling to the `styles` folder and is located at `~/images` with `~` being `root`.

File: styles/styles.css (hunk 4 above)
Comment (codeTrackLift) [reply styles/styles.css]: Please move the `images` folder out of the `styles` folder.  See [this comment](https://github.com/codepharm/git/pull/6#pullrequestreview-1103154662)

**Comment by nettle790** [comment]
please cite one good reason as to why the images folder should be in the root directory??

**Comment by codeTrackLift** [comment]
> please cite one good reason as to why the images folder should be in the root directory??

The images should not be under `styles` as they are not styles. To clarify, I meant the project root, not your local root.

**Comment by nettle790** [comment]
I know what a sibling folder and a tilde is meant for but I am not able to implement it practically.please review to note whether the folder structure is what it should be.

**Comment by codeTrackLift** [comment]
```
.    
├── ...
├── styles                 
│   └── styles.css     
├── images
│   └── filename.jpg        # this can be whatever your image is called
├── .gitignore
├── index.html          
├── LICENSE     
└── README.md
```

Please remove the `root` follow the suggested folder structure above.

File: index.html [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
-    <body>
-        <h1>Hello World!</h1>
-    </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="base/styles/styles.css" type="text/css">
```
Comment (codeTrackLift) [inline index.html]: Thanks for making the file structure updates.  I think we may need to update the `href` here as well.  Suggest to replace `base` with `./`

**Comment by nettle790** [comment]
please review and send me links to  any docs on relative file paths.so far I have been using the text editor right click relative file path option.this is a bit confusing.

**Review by codeTrackLift — APPROVED** [review]

**Comment by nettle790** [comment]
please review the new file structure.is it the right way to create it?
Screenshot 2022-10-07 at 5 18 03 PM

for example in the above file path could you please  reply as to which is the:

1. root directory
2. home directory 
3. project root.

**Comment by codeTrackLift** [comment]
It looks fine, but anything beyond the project folder structure is up to you as that's on your local machine.  

For the changes we did here, I would just propose to keep file type and folder names aligned, an example is [this file structure model](https://github.com/codepharm/git/pull/6#issuecomment-1259900459).

**Comment by nettle790** [comment]
thanks so much.Learning filepaths now  which is important though you left one question unanswered.
Please check why index.html is conflicting in new pull request. I had deleted it and added it again in the project folder.should not be causing an issue though????

**Comment by codeTrackLift** [comment]
It looks like a `project` directory was added and the content in `index.html` was deleted [here](https://github.com/codepharm/git/pull/11/files)

This "git practice" repo is intended for folks to practice `git` commands.

Feel free to continue building things out and take your project as far as you'd like to do, but it's probably not necessary to to merge in this 'practice' repo.  I'm going to close #11 to resolve the conflict, let me know with any questions/concerns.

**Comment by nettle790** [comment]
sorry ,was practicing PR's.

**Pull request merged**